packages/scss/package.json

```diff
@@ -29,7 +29,14 @@
   },
   "homepage": "https://github.com/LuccaSA/lucca-front#readme",
   "devDependencies": {
+    "browser-sync": "^2.18.8",
+    "gulp": "^3.9.1",
+    "gulp-autoprefixer": "^3.1.1",
+    "gulp-clean": "^0.3.2",
+    "gulp-rename": "^1.2.2",
+    "gulp-sass": "^3.1.0",
+    "gulp-stylelint": "^3.9.0",
     "node-sass": "^4.5.2",
     "nodemon": "^1.11.0"
```

Comment:
`nodemon` is not usefull anymore. Does `node-sass` is still needed as it is a dependancy of `gulp-sass` ?